### ISSUE TITLE
Is learning rate decay a regularization technique?

### ISSUE THREAD

**Issue by shuaibaslam2019** [keras-team/keras-io#369]
Upto my understanding, it is a regularization technique, because it helps to learn model correctly and in generalization. But I am still confused at whether it would be correct or not to call it a "regularization" method.??

Thank you!

**Comment by SuryanarayanaY** [comment]
Hi @shuaibaslam2019 ,

Learning rate decay is used basically to avoid noise in convergence and helps convergence smoother and also it makes model both optimized and generalised.

Regularisation plays role in cost/loss function whereas learning rate plays role in weights adjustment and IMO both are not same.
